_docs/tasks/istio-addons.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Using the Istio Add-ons
+
+overview: This task shows you how to setup and use the Istio Addons, including Prometheus, Grafana, and Servicegraph.
+
+order: 120
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to install and use the set of Istio add-ons. The add-ons
+include a Prometheus server with config to pull metrics from Istio endpoints, a
+Grafana server preconfigured with a default dashboard for monitoring an Istio
+service mesh, and an example ServiceGraph service that can generate a visual
+representation of an Istio service mesh.
+
+The [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application is used
+as the example application throughout this task.
+
+## Before you begin
+* [Install Istio](./installing-istio.html) in your cluster and deploy an
```

Comment:
remove ./ ?

Reply:
good catch. this clued me into the fact that we fundamentally changed the structure of the docs repo recently.

Reply:
yeah it keeps changing...  make sure "rake test" is happy and/or navigation works when using your local jekyll

ditto for how the images render